dubbo-configcenter/dubbo-configcenter-nacos/src/main/java/org/apache/dubbo/configcenter/support/nacos/NacosDynamicConfiguration.java

```diff
@@ -157,48 +134,68 @@ private void putPropertyIfAbsent(URL url, Properties properties, String property
      * @return
      */
     private NacosConfigListener createTargetListener(String key, String group) {
+        String normalizedKey = normalizedKey(key);
```

Comment:
duplicate call of `normalizedKey(key)`, the caller of createTargetListener() has already called the same method, so the key passed in is a normalized one.

Reply:
Fixed.